run_autogvp.sh

```diff
@@ -75,27 +95,63 @@ while [ $# -gt 0 ]; do
   shift
 done
 
-## default files
-
-variant_summary_file="$BASEDIR/data/variant_summary.txt.gz"
-submission_summary_file="$BASEDIR/data/submission_summary.txt.gz"
 
-if [[ ! -f $variant_summary_file || ! -f $submission_summary_file ]] ; then
-    echo "Warning: ClinVar variant and/or submission summary files not found. Downloading to data/..."
+# If selected ClinVar submissions files not provided, then run select-ClinVar-submissions.R
+if [[ $selected_submissions -eq 0 ]]; then
     
-    bash $BASEDIR/scripts/download_db_files.sh
-
-fi
-
-select_submissions_file="$BASEDIR/results/ClinVar-selected-submissions.tsv"
-
-if [[ ! -f $select_submissions_file ]] ; then
-    echo "ClinVar-selected-submissions.tsv file not found. Running select-clinvar-submissions.R script..."
+    echo "select ClinVar submission file not specified. Running select-ClinVar-submissions Rscript..."
     
-    Rscript $BASEDIR/scripts/select-clinVar-submissions.R --variant_summary $BASEDIR/data/variant_summary.txt.gz --submission_summary $BASEDIR/data/submission_summary.txt.gz --outdir $out_dir
-
+  # if variant_summary or submission_summary args not provided, check if files exist in data/
+  if [[ $variant_summary -eq 0  || $submission_summary -eq 0 ]]
```

Comment:
```suggestion
  if [[  ! e $variant_summary  || ! e $submission_summary  ]]
```

I was getting this error, which prompted me to suggest changing all of these ifelses for presence of files since they are looking for files and not numerical values, so this is a better way to write it

```
bash run_autogvp.sh --workflow="cavatica" \
--vcf=data/test_pbta.single.vqsr.filtered.vep_105.vcf \
--filter_criteria='FORMAT/DP>=10 (FORMAT/AD[0:1-])/(FORMAT/DP)>=0.2 (gnomad_3_1_1_AF_non_cancer<0.001|gnomad_3_1_1_AF_non_cancer=".")' \
--intervar=data/test_pbta.hg38_multianno.txt.intervar \
--multianno=data/test_pbta.hg38_multianno.txt \
--autopvs1=data/test_pbta.autopvs1.tsv \
--conceptIDs=data/clinvar_cpg_concept_ids.txt \
--conflict_res="most_severe" \
--outdir=results \
--out="test_pbta"
.
select ClinVar submission file not specified. Running select-ClinVar-submissions Rscript...
variant summary and/or submission_summary file(s) not specified. Checking if files exist in data/...
variant and submission summary files found. Running select-ClinVar-submissions Rscript...
run_autogvp.sh: line 132: [[: data/clinvar_cpg_concept_ids.txt: syntax error: invalid arithmetic operator (error token is ".txt")
run_autogvp.sh: line 138: [[: data/clinvar_cpg_concept_ids.txt: syntax error: invalid arithmetic operator (error token is ".txt")
```

Reply:
If looking for presence of file, add `-f` in front, if looking for absence, add `! -e` in front, courtesy of chatgpt ;). Just double check the replacements.